ethnode/configure.yml

```diff
@@ -0,0 +1,143 @@
+- hosts:
+   - "{{ ethnode }}"
+  gather_facts: no
+  vars_files:
+    - ../containers/versions.yml
+  vars:
+    ethnode_user:    "ethnode_user"
+    blockchain_path: "/home/{{ ethnode_user }}/blockchain"
+  tasks:
+    - become:      yes
+      become_user: root
+      block:
+        - fail: msg="Please supply a value for ethnode variable. Valid values are{{ ":" }} 'ethnode-testnet' and 'ethnode-mainnet'"
+          when: ethnode not in ["ethnode-testnet", "ethnode-mainnet"]
+
+        - name: Gathering Facts
+          setup:
+
+        - set_fact:
+            ethnode_config:
+              ethnode-testnet:
+                geth_extra_flags: "--rinkeby"
+                ethereum_chain:   "rinkeby"
+              ethnode-mainnet:
+                geth_extra_flags: ""
+                ethereum_chain:   "frontier"
+
+        - name:   Create a ext4 filesystem on /dev/sdb and check disk blocks
+          filesystem:
+            fstype: ext4
+            dev:    /dev/sdb
+
+        - name:   Install required packages for apt_key and apt_repository
+          apt:
+            name:
+              - apt-transport-https
+              - ca-certificates
+              - curl
+
+        - name:   Add Docker repository key
+          apt_key:
+            url:   https://apt.dockerproject.org/gpg
+            state: present
```

Comment:
Adding docker key/repository could also be extracted into a separate playbook, just like `nstall-basic-utilities.yml`.

Reply:
This is in a separate playbook - `prerequire-configure-tasks.yml`